.gitattributes

```diff
@@ -0,0 +1,2 @@
+# https://stackoverflow.com/a/13264117/9068029
+CHANGELOG.md merge=union
```

Comment:
This is a cool thing instructing git to do a merge union in case of conflicts

Reply:
tested it in this PR when I ran update from master and indeed it fast forwarded and merged the file as expected